backend/config/application.rb

```diff
@@ -15,6 +15,8 @@ class Application < Rails::Application
     config.i18n.default_locale = :en
     config.i18n.available_locales = %i[en ru]
     config.active_model.i18n_customize_full_message = true
+
+    routes.default_url_options[:host] = 'localhost:3000'
```

Comment:
why not 3001?

Reply:
because frontend work on 3001 port, backend work on 3000. In application.rb we configure backend, not frontend